### ISSUE TITLE
Is the first example in README.md correct?

### ISSUE THREAD

**Issue by martinsik** [zont/gulp-bower#47]
Hey, the first example says:

> ```javascript
var gulp = require('gulp');
var bower = require('gulp-bower');

> gulp.task('bower', function() {
  return bower()
    .pipe(gulp.dest('lib/'))
});
```

> This defaults to the directory configured in `./.bowerrc` or to `./bower_components` when no `.bowerrc` could be found.

When you run it with existing `.bowerrc` with for example:

```
{
  "directory": "build/bower_components"
}
```
it ignores the `directory` setting and outputs everything into `lib` directory which is not what I'd expect. Maybe the first example should be just a simple `bower()` call:

```javascript
gulp.task('bower', function() {
  return bower();
});
```

Right now, it seems to me like the first two examples do exactly the same. I actually don't understand what's the difference either.

**Comment by FabianMeul** [comment]
I've also been wondering about this.
It seems you are correct, as your edited example show the correct behaviour.

**Comment by zont** [comment]
Fixed in  #48
